.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We do not commit personal IDE settings

Reply:
To ignore it, add it to your global gitignore file
Related: https://gitlab.com/gitlab-org/gitlab-foss/-/merge_requests/17017